docs/src/userguide/formatter.md

```diff
@@ -1 +1,29 @@
 # Formatting Code
+
+Formatting helps to keep code readable by automatically aligning indentations and spaces.
+
+The Julia formatter can automatically make this code:
+```julia
+f(x)=2x+3
+print(f'( 2 ))
+
+open("myfile.txt", "w") do io
+	write(io, "Hello world!")
+    end;
+```
+
+look like this:
+```julia
+f(x) = 2x + 3
+print(f'(2))
+
+open("myfile.txt", "w") do io
+	write(io, "Hello world!")
+end;
+```
+
+It's the very same code, though now it's much easier to read.
+
+In order to format your code press `shift + cmd|windows + p` to bring up the command palette and search for `Format Document`
```

Comment:
I think `Ctrl-Shift-I` is the default keybinding for formating a document, right?

Reply:
I agree, though it did not work for me, and I couldnt figure out why (it was even shown to be that combination for me, oddly enough).

What would you say is the right thing to do in the docs then?